src/iroh.udl

```diff
@@ -32,13 +32,17 @@ interface Doc {
   [Throws=IrohError]
   DocTicket share_read();
   [Throws=IrohError]
-  SignedEntry set_bytes(AuthorId author, bytes key, bytes value);
+  Hash set_bytes(AuthorId author, bytes key, bytes value);
   [Throws=IrohError]
-  bytes get_content_bytes(SignedEntry entry);
+  bytes get_content_bytes(Hash hash);
   [Throws=IrohError]
-  sequence<SignedEntry> latest();
+  sequence<Entry> latest();
   [Throws=IrohError]
   void subscribe(SubscribeCallback cb);
+  [Throws=IrohError]
+  void stop_sync();
```

Comment:
I was expecting `stop_sync` to take a document ID argument so I can manage sync on multiple documents. That can totally be a thing for later though. Having this at all is a huge win

Reply:
this is a method on the document, so it is already scoped

Reply:
OoooOOOOOOOOooOh, I didn't realize. Still learning how to read this `udl` dialect